editoast/src/main.rs

```diff
@@ -161,3 +165,41 @@ async fn generate(
     }
     Ok(())
 }
+
+// Run the clear subcommand
+// This command clear all generated data for the given infra
```

Comment:
This is not documentation: https://doc.rust-lang.org/book/ch14-02-publishing-to-crates-io.html